decoder/src/lib.rs

```diff
@@ -873,6 +876,20 @@ impl<'t, 'b> Decoder<'t, 'b> {
                     let c = std::char::from_u32(data).ok_or(DecodeError::Malformed)?;
                     args.push(Arg::Char(c));
                 }
+                Type::Debug | Type::Display => {
+                    // UTF-8 stream without a prefix length, terminated with `0xFF`.
```

Comment:
"isn't a 0xFF delimiter problematic with characters like BOM / U+FEFF?"
"no, because UTF-8 encoding never uses the 0xff value; each byte (after encoding) contains at least one zero bit"
case in point, BOM is encoded as `[239, 187, 191]`